examples/generic_connector_configs/README.md

```diff
@@ -217,6 +222,66 @@ to the backend server uses SSL.
 
 ___
 
+### Twitter API
+This example can be used to interact with
+[Twitter's API](https://developer.twitter.com/en/docs).
+
+The configuration file for the Twitter API can be found at
+[twitter_secretless.yml](./twitter_secretless.yml).
+
+**Note:** This configuration currently only supports connecting to the
+Twitter API via OAuth2. An issue can be found
+[here](https://github.com/cyberark/secretless-broker/issues/1297)
+for adding an OAuth1 Connector for Twitter.
+
+#### How to use this connector
+* Edit the supplied service configuration to get your
+[OAuth token](https://developer.twitter.com/en/docs/basics/authentication/oauth-2-0/bearer-tokens)
+* Run secretless with the supplied configuration(s)
+* Query the API using `http_proxy=localhost:8051 curl api.twitter.com/{Request}`
+
+#### Example Usage
+<details>
+  <summary><b>Example setup to try this out locally</b></summary>
+  <ol>
+    <li>
+      Get your
+      <a href="https://developer.twitter.com/en/apps">
+        Twitter API key and Secret Key
+      </a>
+    </li>
+    <li>
+      Get an
+      <a href="https://developer.twitter.com/en/docs/basics/authentication/oauth-2-0/bearer-tokens">
+        OAuth token
+      </a>
+      from Twitter through CURL
+    </li>
+    <code>
+      curl -u 'API key:API secret key' \
+      <br />
+      --data 'grant_type=client_credentials' \
+      <br />
+      'https://api.twitter.com/oauth2/token'
+    </code>
+    <li>Save the local token from the request into the OSX keychain</li>
```

Comment:
Can we provide a better instruction on how to do this? I believe summon needs a specific syntax. We should also specify an alternative for someone who isn't on MacOS, if possible.

Maybe something like "Store your token from the request so that it may be retrieved by the provider you specified in your `secretless.yml`"